src/PageTypes/EventHolderController.php

```diff
@@ -16,6 +16,8 @@
  *
  * When the user clicks on a month, pagination will be reset, but tags retained. Pagination retains all other
  * parameters.
+ *
+ * @extends DatedUpdateHolderController<NewsHolder>
```

Comment:
```suggestion
 * @extends DatedUpdateHolderController<EventHolder>
```
? I'm not sure if this is correct or not

Reply:
Oops! Yup good catch. That was a copy paste error.
Fixed.